plugins/plugin-title/src/index.js

```diff
@@ -92,6 +92,14 @@ export default {
           .attr('data-text-class', 'fill-current font-bold')
           .attr('data-text-transform', transform(so.at.shift(-90 - so.rotation, shift * so.scale)))
       }
+      shift += 8
+      const dateformat = require('dateformat')
+      const now = new Date()
+      this.points[`_${prefix}_exportDate`] = so.at
+        .shift(-90 - so.rotation, shift * so.scale)
+        .attr('data-text', dateformat(now, 'yyyymmdd"T"HHMMo'))
+        .attr('data-text-class', 'fill-note')
```

Comment:
Could we make that class `text-sm` so it's a bit smaller and just black?

Reply:
Change made.